src/pages/Login/Login.scss

```
@@ -0,0 +1,91 @@
+@import '../../styles/variables.scss';
+
+.login {
+  @include flexCenter;
+  height: 100vh;
+
+  .leftWrap {
+    margin-right: 40px;
+
+    .title {
+      color: $color-black;
+      font-size: 25px;
+      letter-spacing: -1px;
+    }
+    .subtitle {
+      margin-top: 10px;
+      color: $color-middle-gray;
+      font-size: 13px;
+      line-height: 16px;
+    }
+    .imagesWrap img {
+      width: 500px;
+    }
+  }
+
+  .rightWrap {
+    padding: 70px;
+    border: 1px solid $color-light-gray;
+
+    .loginTitle {
+      margin-bottom: 30px;
+      font-size: 32px;
+      letter-spacing: -1px;
+    }
+    .loginWrap {
+      width: 280px;
+      display: flex;
+      flex-direction: column;
+
+      .loginId,
+      .loginPassword {
+        padding: 15px 0;
+        border-bottom: 2px solid $color-light-gray;
+        transition: 0.2s;
+      }
+
+      .loginIdInnerText,
+      .loginPasswordInnerText {
+        padding: 15px 0;
+        border-bottom: 2px solid $color-middle-gray;
+        transition: 0.2s;
+      }
```

1. 동일한 css를 가지는 element라면 동일한 className 하나로 사용할 수 있을 것 같습니다.
2. 바뀌는 부분만 따로 className 작성하는 방식으로 로직 짜볼 수 있겠네요!

```suggestion
      .loginInput {
        padding: 15px 0;
        border-bottom: 2px solid $color-light-gray;
        transition: 0.2s;
      }

      &.underline {
        border-bottom: 2px solid $color-middle-gray;
      }
```

---

넵 감사합니다!
클래스를 세세하게 나누면 좋은줄 알았어요 수정할게요!